### PR TITLE
Add `ig` documentation for some trace tools

### PR DESCRIPTION
**Pull request by eiffel-fl**
Hi.

This PR removes the TODOs regarding `ig` documentation for `trace` tools.

Best regards.

### CODE REVIEW

**Review by alban — DISMISSED** [review]
Thanks! A few things to fix and some suggestions. It should be good after that.

File: docs/gadgets/trace/dns.md [hunk 1]
```diff
@@ -58,7 +58,37 @@ namespace "demo" deleted
...
 
-TODO
+Let's start the gadget in a terminal:
+
+```bash
+$ sudo ig trace dns -c test-trace-dns
+RUNTIME.CONTAINERNAME                   PID        TID        COMM             QR TYPE      QTYPE      NAME                                   RCODE                NUMANSW…
+```
+
+Launch a container that makes DNS requests
```
Comment (alban) [inline docs/gadgets/trace/dns.md:68]:
```suggestion
Launch a container that makes DNS requests:
```

(sorry, I missed that in my previous comment)

File: docs/gadgets/trace/fsslower.md [hunk 2]
```diff
@@ -71,4 +71,38 @@ $ kubectl delete pod mypod
...
+$ docker run --name test-trace-fsslower -it --rm debian /bin/sh -c "apt-get update && apt-get install -y git"
+Get:1 http://deb.debian.org/debian bullseye InRelease [116 kB]
+Get:2 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
+...
+0 added, 0 removed; done.
+Running hooks in /etc/ca-certificates/update.d...
+done.
+```
+
+The tool will list the I/O operations that were slower than 1ms::
```
Comment (alban) [inline docs/gadgets/trace/fsslower.md:93]:
```suggestion
The tool will list the I/O operations that were slower than 1ms:
```

(sorry, the mistake comes from my previous review)

File: docs/gadgets/trace/oomkill.md [hunk 3]
```diff
@@ -69,4 +69,23 @@ pod "test-pod" deleted
...
+
+```bash
+$ docker run --name test-trace-oomkill -m 512M -it --rm busybox tail /dev/zero
+```
+
+The tool will show the killed process:
+
+```bash
+$ sudo ig trace oomkill -c test-trace-oomkill
+RUNTIME.CONTAINERNAME                                                                           KPID       KCOMM            PAGES               TPID       TCOMM
```
Comment (alban) [inline docs/gadgets/trace/oomkill.md:89]: Why are the field kpid, kcomm, etc. empty?

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**